### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hadrien-toma**

### CODE REVIEW

**Comment by broerse** [comment]
@hadrien-toma It both seems to work. See

https://github.com/pouchdb-community/pouchdb-authentication/blob/cd12d064e66848f1d8a56db58838d6ef5853f39f/src/index.js#L17

So do you think this is the preferred way to write it?

**Comment by hadrien-toma** [comment]
@broerse, as it is the way the [API documentation](https://github.com/pouchdb-community/pouchdb-authentication/blob/master/docs/api.md) describes it, yes :wink:.

**Comment by ptitjes** [comment]
@broerse @hadrien-toma Yes, it's so that it is aligned with PouchDB Server's `pouchdb-auth`. The form without caps is supposed to be deprecated at some point.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Easy user authentication for PouchDB/CouchDB.
 
 ```js
 var db = new PouchDB('http://mysite:5984/mydb', {skip_setup: true});
-db.login('batman', 'brucewayne').then(function (batman) {
+db.logIn('batman', 'brucewayne').then(function (batman) {
   console.log("I'm Batman.");
   return db.logout();
```
Comment (ptitjes) [inline README.md]: Would you mind modifying also the `logOut` call ?

Comment (hadrien-toma) [reply README.md]: Sure! I think this is done now :smiley: !

**Comment by broerse** [comment]
@hadrien-toma Thanks!

**Comment by ptitjes** [comment]
@broerse, just a note to ask you to **not use the big green button** to merge pull requests. I'd prefer that we use the same workflow PouchDB commiters use, as it avoid merge commits (see How to merge a pull request in https://github.com/pouchdb/pouchdb/blob/master/CONTRIBUTING.md#committers). I still have to add some explanations about that in a `CONTRIBUTING.md` file.

**Pull request merged**